.github/workflows/sycl-docs.yml

```
@@ -25,16 +25,14 @@ jobs:
         path: repo
     - name: Install deps
       run: |
-        sudo apt-get install -y doxygen graphviz ssh ninja-build libhwloc-dev
+        sudo apt-get install -y graphviz ssh ninja-build libhwloc-dev
```

```suggestion
        sudo apt-get install -y ssh ninja-build libhwloc-dev
```
Do we need graphviz after Doxygen removal?

---

Yes, we still need graphviz. Otherwise I get the following error (https://github.com/uditagarwal97/llvm/actions/runs/11018898111/job/30600198171?pr=23#step:4:996):

```
cd /home/runner/work/llvm/llvm/build/tools/clang/docs && /usr/local/bin/cmake -E env /usr/local/bin/sphinx-build -b html -d /home/runner/work/llvm/llvm/build/tools/clang/docs/_doctrees-clang-html -q -t builder-html -D version=20 -D release=20.0.0git -tPreRelease -W /home/runner/work/llvm/llvm/build/tools/clang/docs /home/runner/work/llvm/llvm/build/tools/clang/docs/html

Warning, treated as error:
dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
```